### PR TITLE
Update the CONTRIBUTING.md according to feedbacks from Javier

### LINKED ISSUE #19: Fix Contributing Document
Addressing #18, delete Python parts and edit it for R

### PR DESCRIPTION
**Pull request by YueJiangMDSV**
Hi Team,

I updated the CONTRIBUTING.md according to feedbacks from Javier. Please review it, and double check if everything is fine. Let me know if there is anything needs update. Thanks!

Alex

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ and "help wanted" is open to whoever wants to implement it.
...
-samplingsimulatorpy could always use more documentation, whether as part of the
-official samplingsimulatorpy docs, in docstrings, or even on the web in blog posts,
+samplingsimulatorr could always use more documentation, whether as part of the
+official samplingsimulatorr docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
 ### Submit Feedback
 
-The best way to send feedback is to file an issue at https://github.com/UBC-MDS/samplingsimulatorr/issues.
+The best way to send feedback is to file an issue at [here](https://github.com/UBC-MDS/samplingsimulatorr/issues).
```
Comment (hwilliams10) [inline CONTRIBUTING.md]: Suggest deleting "at".
```suggestion
The best way to send feedback is to file an issue [here](https://github.com/UBC-MDS/samplingsimulatorr/issues).
```

**Review by hwilliams10 — CHANGES_REQUESTED** [review]
Only one small typo. Please check my suggestion. 
Otherwise looks great!

File: CONTRIBUTING.md (hunk 1 above)
Comment (YueJiangMDSV) [reply CONTRIBUTING.md]: Thanks for pointing out! I just fixed it. Let me know if it needs additional changes.

**Review by hwilliams10 — APPROVED** [review]
Thanks for that!

**Comment by hwilliams10** [comment]
resolves #19

**Pull request merged**